docs/core/diagnostics/diagnostic-port.md

```diff
@@ -1,67 +1,69 @@
+`{pid}` is the process id written in decimal, `{temp}` is the `TMPDIR` environment variable or the value `/tmp` if `TMPDIR` is undefined/empty, and `{disambiguation_key}` is the process start time written in decimal. On macOS and NetBSD, the process start time is number of seconds since UNIX epoch time. On all other platforms, it's jiffies since boot time.
 
 ## Suspend the runtime at startup
 
-By default, the runtime executes managed code as soon as it starts, regardless whether any diagnostic tools have connected to the diagnostic port. Sometimes it is useful to have the runtime wait to run managed code until after a diagnostic tool is connected, to observe the initial program behavior. Setting environment variable `DOTNET_DefaultDiagnosticPortSuspend=1` causes the runtime to wait until a tool connects to the default port. If no tool is attached after several seconds, the runtime prints a warning message to the console explaining that it is still waiting for a tool to attach.
+By default, the runtime executes managed code as soon as it starts, regardless of whether any diagnostic tools have connected to the diagnostic port. Sometimes it's useful to have the runtime wait to run managed code until after a diagnostic tool is connected, to observe the initial program behavior. Setting environment variable `DOTNET_DefaultDiagnosticPortSuspend=1` causes the runtime to wait until a tool connects to the default port. If no tool is attached after several seconds, the runtime prints a warning message to the console explaining that it's still waiting for a tool to attach.
 
 ## Configure additional diagnostic ports
 
-> [!IMPORTANT]
+> [!NOTE]
 > This works for apps running .NET 5 or later only.
 
-Both the Mono and CoreCLR runtimes can use custom configured diagnostic ports. These ports are in addition to the default port that remains available. There are a few common reasons this is useful:
+Both the Mono and CoreCLR runtimes can use custom configured diagnostic ports in the `connect` role. These ports are in addition to the default port that remains available. There are a few common reasons this is useful:
 
-- On Android, iOS and tvOS there is no default port, so configuring a port is necessary to use diagnostic tools.
-- In environments with containers or firewalls, you may want to set up a predictable endpoint address that doesn't vary based on process id as the default port does. Then the custom port can be explicitly added to an allow list or proxied across some security boundary.
-- For monitoring tools it is useful to have the tool listen on an endpoint, and the runtime actively attempts to connect to it. This avoids needing the monitoring tool to continuously poll for new apps starting. In environments where the default diagnostic port isn't accessible, it also avoids needing to configure the monitor with a custom endpoint for each monitored app.
```

Comment:
This paragraph still holds true - the runtime still has connect mode such hat the monitor tool can just listen on a port rather than connect-loop poll.